schema/make_schema.py

```diff
@@ -156,21 +147,25 @@ def format_param(param: click.core.Option) -> SamCliParameterSchema:
     """
     if not param:
         raise SchemaGenerationException("Expected to format a parameter that doesn't exist")
-    if not param.type.name and param.name not in EXCEPTION_OPTION_CLASS_MAPPER.keys():
+    if not param.type.name:
         raise SchemaGenerationException(f"Parameter {param} passed without a type:\n{param.type}")
 
-    param_type = param.type.name.lower()
+    param_type = []
```

Comment:
Do we need to update unit tests to test these changes?

Reply:
Including them may be beneficial, yea, since we can verify the behaviour of having multiple allowable types as well 👍